Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/EditorMaterialSystemComponent.cpp

```diff
@@ -241,7 +241,14 @@ namespace AZ
 
         void EditorMaterialSystemComponent::OnPopulateToolMenuItems()
         {
-            if (!m_openMaterialEditorAction)
+            // Removing standalone "Material Editor" option in editor's Tools menu now that O3DEMaterialEditor gem
+            // provides an integrated version.
+            // 
+            // TODO: It's probably a good idea to move AtomMaterialEditor (the integrated version of atom material editor)
+            //       from MaterialEditor Gem to this AtomLyIntegration_CommonFeatures gem, as it's the gem that integrates
+            //       atom features into o3de.
+            //
+            /*if (!m_openMaterialEditorAction)
```

Comment:
@gadams3 I just realized that this file `EditorMaterialSystemCompoment.cpp` could do the registration of Atom Material Editor window. That way the `AtomMaterialEditor` and `AtomMaterialEditor.Static` targets can be removed from `MaterialEditor` gem. This cpp could use the intermediate static library of the documents, viewport, windows classes we talked about in other comments (`MaterialEditor.Static`). It seems like a great fit because it's in AtomLyIntegration where it's added the integrated version of the material editor. Any thoughts?